targets/TARGET_STM/rtc_api_hal.h

```diff
@@ -33,19 +33,18 @@
 
 #include <stdint.h>
 #include "rtc_api.h"
+#include "ticker_api.h"
+#include "lp_ticker_api.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
-/*
- * Extend rtc_api.h
- */
 
-/** Set the given function as handler of wakeup timer event.
- *
- * @param handler    The function to set as handler
- */
-void rtc_set_irq_handler(uint32_t handler);
+#if RTC_LSI
```

Comment:
be more defensive, what values can it have, any non zero is LSI and zero LSI, what if its not defined?